ui/mods/msu/mod_settings/settings_screen.js

```diff
@@ -98,6 +98,12 @@ ModSettingsScreen.prototype.createDIV = function (_parentDiv)
 	this.mModPageScrollContainer = this.mModPageContainer.findListScrollContainer();
 };
 
+ModSettingsScreen.prototype.bindTooltips = function ()
+{
+	this.mDialogContainer.find('.main-cancel-button').bindTooltip({ contentType: 'ui-element', elementId: MSU.TooltipIdentifiers.ModSettings.Main.Cancel});
+	this.mDialogContainer.find('.main-save-button').bindTooltip({ contentType: 'ui-element', elementId: MSU.TooltipIdentifiers.ModSettings.Main.OK});
+};
+
```

Comment:
Why not just create variables for these?

Reply:
Fine for now